src/components/onboarding/steps/budget.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import SelectableToggle from "../selectable_toggle"
+import { StepProps } from "../types"
+import { Layout } from "./layout"
+
+const OptionsContainer = styled.div`
+width: 450px;
+margin: 0 auto 100px;
+&:last-child {
+  border-bottom: 1px solid #e5e5e5;
+}
+`
+
+interface State {
+  selection: string
+}
+
+export default class Budget extends React.Component<StepProps, State> {
+  options = [
+    "UNDER $500",
+    "UNDER $2,500",
+    "UNDER $5,000",
+    "UNDER $10,000",
+    "UNDER $25,000",
+    "UNDER $50,000",
+    "NO BUDGET IN MIND",
+  ]
```

Comment:
Nit: This is an instance variable, whereas this could just be a static constant outside the class.

Reply:
In the case of a static constant outside the class, what does that look like? Does it then change to be some sort of options prop that get's pass in via the constructor function?